windows/build_lld.bat

```diff
@@ -0,0 +1,23 @@
+@setlocal
+
+set LLVM_BRANCH=master
+
+call "c:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvars32.bat"
+git clone --depth 1 --branch %LLVM_BRANCH% https://github.com/llvm-mirror/llvm.git llvm || exit /B 1
+git clone --depth 1 --branch %LLVM_BRANCH% https://github.com/llvm-mirror/lld.git llvm\tools\lld || exit /B 1

```

Comment:
I'd rather go with the released sources, then we also have proper versioning.
http://releases.llvm.org/download.html#5.0.1

http://releases.llvm.org/5.0.1/lld-5.0.1.src.tar.xz
http://releases.llvm.org/5.0.1/llvm-5.0.1.src.tar.xz

Bonus, check gpg signatures locally and put the sha256sums in the script (prolly simpler than gpg on window).

```
d5b36c0005824f07ab093616bdff247f3da817cae2c51371e1d1473af717d895  lld-5.0.1.src.tar.xz
5fa7489fc0225b11821cab0362f5813a05f2bcf2533e8a4ea9c9c860168807b0  llvm-5.0.1.src.tar.xz
```

Reply:
Changed to download the tarballs. How would you verify the SHA? (I don't think this is super important, though.)